pkg/reconciler/kubernetes/initcontroller/initcontroller.go

```diff
@@ -41,6 +42,16 @@ type PayloadOptions struct {
 	ReadOnly bool
 }
 
+func OperatorVersion(ctx context.Context) (string, error) {
```

Comment:
Can't we read this from the config map ?

Reply:
yes but the version is already in the env of pod so easier that way
why to do an API call.. :)
also configmap is for other external tools to know the version of operator